src/server/stream_family_test.cc

```diff
@@ -499,4 +499,54 @@ TEST_F(StreamFamilyTest, XTrimInvalidArgs) {
   EXPECT_THAT(resp, ErrArg("syntax error"));
 }
 
+TEST_F(StreamFamilyTest, Xclaim) {
+  Run({"xadd", "foo", "1-0", "k1", "v1"});
+  Run({"xadd", "foo", "1-1", "k2", "v2"});
+  Run({"xadd", "foo", "1-2", "k3", "v3"});
+  Run({"xadd", "foo", "1-3", "k4", "v4"});
+
+  // create a group for foo stream
+  Run({"xgroup", "create", "foo", "group", "0"});
+  // alice consume all the stream entries
+  Run({"xreadgroup", "group", "group", "alice", "streams", "foo", ">"});
+
+  // bob claims alice's two pending stream entries
```

Comment:
`XCLAIM` is kinda useless without `XPENDING`, right?
I mean, how would bob know which messages IDs to takeover?

Reply:
Honestly, yes. But `XPENDING` is not supported yet :).

Reply:
Sounds like a good next step then :)
(doesn't have to be you, necessarily)